libs/agno/agno/agent/agent.py

```diff
@@ -330,6 +330,8 @@ class Agent:
     # Stream the response from the Agent
     stream: Optional[bool] = None
     # Stream the intermediate steps from the Agent
+    stream_events: bool = False
```

Comment:
Rather make stream_events None, then make the value you actually set in the init only `stream_events`, but use either `stream_intermediate_steps` or `stream_events`, depending which one is set.

That way you can, internally, only use `stream_events`.